docs/community/release_notes/release_note_0.4.0.md

```diff
@@ -0,0 +1,268 @@
+- [CELEBORN-885][SPARK] Shade RoaringBitmap to avoid dependency conflicts
+- [CELEBORN-875][FOLLOWUP] Enhance `DataPushQueueSuiteJ` for thread safety and prevent `NullPointerException`
+- [CELEBORN-868][MASTER] Adjust logic of SlotsAllocator#offerSlotsLoadAware fallback to roundrobin
+- [CELEBORN-835] Format specifiers should be used instead of string concatenation
+- [CELEBORN-833] Remove unused code
+- [CELEBORN-815] Remove unused ShuffleClient.readPartition
+- [CELEBORN-794] Fix link of CONFIGURATIONS in README
+- [CELEBORN-781] Refactor RPC message type name
+- [CELEBORN-780] Change SPARK_SHUFFLE_FORCE_FALLBACK_PARTITION_THRESHOLD default to Int.MaxValue since slot's is not a bottleneck
+- [CELEBORN-733] Clean unused GetBlacklist & GetBlacklistResponse
+- [CELEBORN-655][SPARK] Rename newAppId to appUniqueId
+
+### BUILD
+
+- [CELEBORN-1205] Disable Maven local caches to improve SBT building stability
+- [CELEBORN-1203] Add `LicenseAndNoticeMergeStrategy` to resolve inner project LICENSE/NOTICE conflict for shaded client packaging
+- [CELEBORN-1199][FOLLOWUP] Disabled the plugin `AddMetaInfLicenseFiles` for shaded clients
+- [CELEBORN-1199] Add LICENSE and NOTICE files for service related sub-projects
+- [CELEBORN-1198] Keep debug info when use SBT build
+- [CELEBORN-1194] Add sbt-pgp plugin for publishing signed artifacts
+- [CELEBORN-1191][FOLLOWUP] Migrate the release script from Maven to SBT
+- [CELEBORN-1191] Migrate the release script from Maven to SBT
+- [CELEBORN-1156][BUILD] SBT publish support
+- [CELEBORN-1159][BUILD] Update the scope of the `protobuf-java` dependency from `protobuf` to `runtime`
+- [CELEBORN-1031] SBT correct the LICENSE and NOTICE for shaded client jars
+- [CELEBORN-1002] Add SBT MRClientProject
+- [CELEBORN-989] Add support for making distribution package via SBT
+- [CELEBORN-913][FOLLOWUP] Recover SBT CI jobs skipped due to last commit
+- [CELEBORN-919][FOLLOWUP] Add SBT project `sparkColumnarShuffle` to `sparkGroup`
+- [CELEBORN-906][FOLLOWUP] Removal of redundant dependency `log4j-slf4j2-impl` from SBT profile `spark-3.4`
+- [CELEBORN-906][BUILD] Aligning dependencies between SBT and Maven
+- [CELEBORN-921] Upgrade sbt to 1.9.4
+- [CELEBORN-898][INFRA] Fix java.lang.NoClassDefFoundError: org/hamcrest/SelfDescribing for SBT CI
+- [CELEBORN-884][BUILD] Consolidate all dependencies into a global object `Dependencies`
+- [CELEBORN-880] Remove sbt compiler plugin `genjavadoc-plugin`
+- [CELEBORN-850][INFRA] Add SBT CI
+- [CELEBORN-843][BUILD] `sbt` support flink-related module build/test
+- [CELEBORN-867][BUILD] Add maven local repository to sbt respositories
+- [CELEBORN-836][BUILD] Initial support sbt
+
+### Documentation
+
+- [CELEBORN-1207] SBT http repository documentation
+- [CELEBORN-1223] Align master and worker metrics of document with MasterSource and WorkerSource
+- [CELEBORN-1247] Output config's alternatives to doc
+- Add the Celeborn Helm Chart doc
```

Comment:
no tikcet?

Reply:
the [PR-2149](https://github.com/apache/incubator-celeborn/pull/2149) was merged by @FMX

Reply:
for such cases, let's add a [MINOR] prefix